cmd/yggd/main.go

```diff
@@ -377,6 +377,9 @@ func main() {
 		env := []string{
 			"YGG_SOCKET_ADDR=unix:" + c.String("socket-addr"),
 			"PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
```

Comment:
@subpop is it good idea to have `PATH` hard-coded like this? Wouldn't be better to get current `PATH` using `os.Getenv("PATH")`?

Reply:
Just to point here, when using os.Getenv("PATH") workers has the following path: `PATH=/sbin:/bin:/usr/sbin:/usr/bin` 
So maybe changing that could cause some issues on current workers that they expect to use local paths.

Reply:
Changing this line is also tangential to the PR in question here, so if we want to adjust the value of `PATH` that's set for workers, we should follow-up in a separate PR.

Reply:
Yeah, I agree that this is not related to this PR. I just wanted to ask, when I saw this line.